lib/pushmi_pullyu/cli.rb

```diff
@@ -123,11 +124,11 @@ def run_tick_loop
     @running = true # set to false by signal trap
 
     while @running
-      sleep(5)
-      logger.debug('.')
       # Preservation (TODO):
       # 1. Montior queue
       # 2. Pop off GenericFile element off queue that are ready to begin process preservation event
+      item = @queue.wait_next_item
```

Comment:
Your missing the initialization of @queue? Currently a nil object.

Can probably throw it in the initialize method?

```ruby
   attr_accessor :config
   attr_accessor :queue

  def initialize
    self.queue = PushmiPullyu::PreservationQueue.new # or pass in some options?
    self.config = PushmiPullyu::Config.new
  end
```

Then use the queue getter here:

```ruby
item = queue.wait_next_item
```
Correct?